pkgs/applications/terminal-emulators/iterm2/default.nix

```
@@ -1,48 +1,35 @@
-{ lib, stdenvNoCC, fetchFromGitHub }:
+{ lib, pkgs, stdenv }:
```

```suggestion
{ lib, stdenv }:
```

Please list the packages individually.